recipe/meta.yaml

```diff
@@ -22,7 +22,7 @@ requirements:
     - python
     - pip
     - wheel
-    - versioneer
+    - versioneer ==0.29
```

Comment:
just wondering why this is necessary? a comment would be handy as it's not immediately obvious

Reply:
Added comment and scoped it for just py312